packages/utils/src/bulk-insert-patients.ts

```diff
@@ -77,6 +78,16 @@ function normalizeName(name: string): string {
   return toTitleCase(name);
 }
 
+function normalizePhone(phone: string): string {
+  if (phone.length === 11 && phone[0] === "1") {
```

Comment:
Maybe `.trim()` before checking for length?

Reply:
yeah good point